### DISCUSSION TITLE
Thread Link

### DISCUSSION THREAD

**Discussion by YassineOsip** [rusq/slackdump#155] [Q&A] [answered]
Hey there,

Is there any flag to aggregate slack thread link with scraped data or not.

thanks,
Yassine

**Comment by rusq** [comment, marked as answer]
Hey @YassineOsip, there's no explicit thread link saved in the json files, but you can reconstruct it.  It looks like:
```
https://<workspace name>.slack.com/archives/<channel_id>/p<message.thread_ts without ".">
```

Example:
```
https://sdump.slack.com/archives/C03EDPUCR53/p1661215809421119
```

You will need to know the slack workspace name.

**Reply by YassineOsip** [reply]
Thanks seems good !
